draft-ietf-rats-corim.md

```diff
@@ -162,19 +162,20 @@ and {{Section G of -cddl}}. Terms and concepts are always referenced as proper n
 
 # Verifier Reconciliation {#sec-verifier-rec}
 
-This specification describes the CoRIM format and documents how a Verifier should process the CoRIM to enable CoRIM authors to convey their intended meaning.
+This specification describes the CoRIM format and documents how a Verifier should process the CoRIM. This enables CoRIM authors, to use CoRIM such that it conveys its intended meaning.
```

Comment:
Prefer to split sentences with a line break. The comma after authors is wrong and the sentence itself is awkward.